extensions/2.0/Vendor/NV_materials_mdl/README.md

```diff
@@ -106,6 +108,7 @@ BSDF measurements (MBSDFs) as defined in Appendix B of the [MDL Language Specifi
 |:---------|:-----|:------------|:---------|
 | `uri` | `string` | The URI (or IRI) of the MBSDF. | No |
 | `bufferView` | `integer` | The index of the buffer view that contains the MBSDF. | No |
+| `mimeType` | `string` | The BSDF measurement's media type. | No |
```

Comment:
Thanks for the suggestion. I also loosened the requirement of the media types for data-URIs. They should match the format as well and don't have to be only `application/vnd.mdl-mbsdf` (or `application/vnd.mdl` for modules).

Reply:
> loosened the requirement of the media types for data-URIs

Then the text should mention that only `application/vnd.mdl-mbsdf` and `application/vnd.mdl` are currently supported. Otherwise, it would imply that any module or BSDF measurements format is allowed as long as data is provided via data-URI.

> When the `bufferView` property is defined, then the `mimeType` property must also be defined and the module format must match the `mimeType` property when the latter is defined.

As currently written, this may imply that `mimeType` must match the content only when `bufferView` is defined. Given that the extension spec (as well as the core glTF 2.0 btw) does not disallow defining `mimeType` together with `uri`, please move this requirement into a separate sentence.

Reply:
> Then the text should mention that only application/vnd.mdl-mbsdf and application/vnd.mdl are currently supported.

We might want to support the NVIDIA Material Definition Language Encapsulated (MDLE) format in the `modules` array in the future. But this would probably only entail adding the correct file extension and media type, but we haven't looked into it in detail yet. Assuming nothing more would have to be added for support, would a separate extension that loosens this sentence then be the way to go?

Reply:
> would a separate extension that loosens this sentence then be the way to go?

If the original MDL extension was a multi-vendor or a Khronos-ratified extension, then the set of supported formats would be restricted within the first extension and potentially expanded in a follow-up extension.

However a single-vendor extension, which likely has only one implementation, could be updated in-place.

Reply:
With that knowledge, I changed the phrasing back to requiring the two media types we have defined so far for data-URIs and we might modify the text later to support the media type for the MDLE format.  Thanks for clarifying and taking the time to review our changes!

Reply:
Any update on this @lexaknyazev?